packages/consumption/src/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.ts

```diff
@@ -67,25 +79,76 @@ export class ProposeAttributeRequestItemProcessor extends GenericRequestItemProc
         requestInfo: LocalRequestInfo
     ): Promise<ValidationResult> {
         const parsedParams: AcceptProposeAttributeRequestItemParameters = AcceptProposeAttributeRequestItemParameters.from(params);
-
-        let attribute = parsedParams.attribute;
+        let attribute;
```

Comment:
Useless declaration, right? `attribute` is declared here, and not used until is reassigned a few lines later.

Reply:
Nevermind, my bad :-)